package/system-role-hpc-server.changes

```diff
@@ -1,3 +1,8 @@
+Wed Apr 12 12:00:00 UTC 2018 - cgoll@suse.com
+
+- Setting minimalistic_libzypp_config to false (bsc#1088275)
+
+-------------------------------------------------------------------
```

Comment:
this is wrong formatting. It should be at first line. I recomment to use `osc vc` for adding changes.